pyhyp/MExt.py

```diff
@@ -5,14 +5,14 @@
 import sys
 
 
-def _tmp_pkg(directory):
+def _tmp_pkg(dirName):
```

Comment:
Why did you change this? `directory` isn't a built-in keyword right? I'm fine with renaming just wanted to understand

Reply:
I changed `dir` to `dirName` at the same time you changed `dir` to `directory`. I guess I won in the merge. I changed it back in any case.

Reply:
I like that better, I'll just remove the last commit and then approve.